### ISSUE TITLE
Silithus Horde Flight Master "Sulhasa"

### ISSUE THREAD

**Issue by Godmatik** [darrowshire-dev/bugtracker#247]
Flight Master Sulhasa - she is standing near the entrance to Silithus, I'm assuming this is due to patch 1.3

However - you can't use it, says you are too far from the taxi stand when you try.

**Comment by Godmatik** [comment]
Update: when you fly to this location from somewhere else, it lands you at Un'goro Marshal's Refuge

**Comment by Daribon** [comment]
I'm the one who spawned Sulhasa in vmangos so I think its adequate that I respond to this.
The reason why you are getting the error "too far away from the taxi stand" is because the server is running 1.12.1. In patch 1.8 and above the flight path node was changed from Valor's Rest to Cenarion Hold, so when you are interacting with the NPC in 1.12.1, it's actually expecting you to be at Cenarion Hold.
![WDBX_Editor_5fpezIPlyg](https://user-images.githubusercontent.com/6137576/129446880-82d6885d-30bf-42e4-8198-44dfa6939131.png)
![WDBX_Editor_uof1D0EZco](https://user-images.githubusercontent.com/6137576/129446883-eee8cb0f-d333-48be-b647-6a4f4669283c.png)
As to why its showing Un'goro I'm not sure, but that flight path was added later in the game as well (it wasn't in 1.3).

So TL;DR, server is running 1.12.1 dbc (DataBaseClient).
For this to work correctly server should use 1.7.1 dbc files or patch the 1.12.1 taxipath.dbc with the

**Comment by Godmatik** [comment]
Good info.

Since the 1.12 FP is now properly spawned in Cenarion Hold (as of today), probably best to remove her from Valor's Rest (she is still here).
